src/api.ts

```diff
@@ -12,7 +12,7 @@ import { assertNever } from '@cmt/util';
 export class CMakeToolsApiImpl implements api.CMakeToolsApi {
     constructor(private readonly manager: ExtensionManager) {}
 
-    version: api.Version = api.Version.v1;
+    version: api.Version = api.Version.v2;
```

Comment:
It's not a huge deal given what I expect is pretty low usage of the API, but we can do better by dynamically selecting which version of the API this claims to be based on the version that was requested from `getApi()` call.

As is, the extension will always return a v2 API object, even if the caller requested v1. Because v2 is backwards-compatible with v1, there should be no harm in simply overriding the API version in the returned object to be v1 when v1 is requested. That will allow any existing exact match queries for the v1 version of the API to continue to succeed even after the 1.17 release.

Reply:
I like this, good idea, I'll modify.

Reply:
Made the changes, let me know if you think I should do it in a different way @benmcmorran . Thanks!

Reply:
@benmcmorran Ping.